### ISSUE TITLE
Path error arises

### ISSUE THREAD

**Issue by weselow** [NeedsCoffee/Tailscale-Updater-Windows#1]
Hi, you did a cool script to automate so nerd task)

I faced the following error: `Coudn't find path "C:\ProgramData\Tailscale-updater\stable", because it is not exists`.

Full log, PowerShell 5:

```
PS C:\Users\59874\Downloads> .\Install-Updater.ps1
Creating directory: C:\ProgramData\Tailscale-updater
Installation target: C:\ProgramData\Tailscale-updater
Successfully installed scheduled task: Tailscale Updater
PS C:\Users\59874\Downloads> cd c:\ProgramData\Tailscale-updater\
PS C:\ProgramData\Tailscale-updater> .\Tailscale-Updater-Windows.ps1
Tailscale release: 1.16.2
Creating directory: C:\ProgramData\Tailscale-updater\tailscale_silo\stable
Release downloaded: C:\ProgramData\Tailscale-updater\tailscale_silo\stable\tailscale-ipn-setup-1.16.2.exe
Starting installation: 'C:\ProgramData\Tailscale-updater\tailscale_silo\stable\tailscale-ipn-setup-1.16.2.exe' /S
Get-ChildItem : Не удается найти путь "C:\ProgramData\Tailscale-updater\stable", так как он не существует.
C:\ProgramData\Tailscale-updater\Tailscale-Updater-Windows.ps1:19 знак:29
+ ... ay]$files = Get-ChildItem -LiteralPath $silo -File -Filter '*.exe' |  ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\ProgramData\Tailscale-updater\stable:String) [Get-ChildItem], ItemNo
   tFoundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand

PS C:\ProgramData\Tailscale-updater> .\Tailscale-Updater-Windows.ps1 -Track stable -DownloadOnly -Verbose
ПОДРОБНО: Release track: stable
ПОДРОБНО: GET with 0-byte payload
ПОДРОБНО: received -1-byte response of content type text/html; charset=utf-8
ПОДРОБНО: Content retrieved: 14746 bytes
ПОДРОБНО: Filtered link: <a href="tailscale-ipn-setup-1.16.2.exe">tailscale-ipn-setup-1.16.2.exe</a>
ПОДРОБНО: Release file: tailscale-ipn-setup-1.16.2.exe
ПОДРОБНО: Release url: https://pkgs.tailscale.com/stable/tailscale-ipn-setup-1.16.2.exe
ПОДРОБНО: Parsed release version: 1.16.2
ПОДРОБНО: Tailscale status: Running
ПОДРОБНО: Tailscale directory: C:\Program Files (x86)\Tailscale IPN
ПОДРОБНО: Tailscale version: 1.16.2.0
No new release found.
ПОДРОБНО: Silo Maintenace: C:\ProgramData\Tailscale-updater\tailscale_silo\stable
Get-ChildItem : Не удается найти путь "C:\ProgramData\Tailscale-updater\stable", так как он не существует.
C:\ProgramData\Tailscale-updater\Tailscale-Updater-Windows.ps1:19 знак:29
+ ... ay]$files = Get-ChildItem -LiteralPath $silo -File -Filter '*.exe' |  ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\ProgramData\Tailscale-updater\stable:String) [Get-ChildItem], ItemNo
   tFoundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand

PS C:\ProgramData\Tailscale-updater>
```

**Comment by NeedsCoffee** [comment]
Great find, thanks @weselow, that looks to be a bug in the silo maintenance function. Thankfully it won't stop the updater from working. Looking into it...

**Comment by NeedsCoffee** [comment]
Thanks again, this is fixed in v0.3.0
